backend/inspirehep/alembic/2d7ea622feda_add_index_to_authors_records.py

```diff
@@ -0,0 +1,35 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""add index to authors_records"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "2d7ea622feda"
+down_revision = "412aeb064d68"
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
+    """Upgrade database."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index(
+        "ix_records_authors_id_type_authors_id",
+        "records_authors",
+        ["id_type", "author_id"],
+        unique=False,
+    )
```

Comment:
you need to create the index concurrently to avoid locking the table, like in https://github.com/inspirehep/inspirehep/blob/cb15e67376042277a0550a2ae0f8b7147a8d65ae/backend/inspirehep/alembic/318758a589d5_modify_idx_object_index_in_pidstore_pid_.py#L21-L29

Reply:
what you did is not sufficient, you need to go into autocommit mode (disable transactions), look at the above snippet.